tests/testBackground.py

```diff
@@ -113,9 +89,9 @@ def testOddSize(self):
 
         for iy, by in zip([image.get(0, y) for y in range(H)],
                           [back.get(0, y) for y in range(H)]):
-            self.assertTrue(abs(iy - by) < 5)
+            self.assertLess(abs(iy - by), 5)
 
-        if False:
+        if display:
```

Comment:
Interesting. The line was added in commit f31e7b32ba20a8b3b3a531361fdeabc24a7245d1 -- does the code work if a display is actually enabled? I'm not sure that this display code is compatible with the intent of the `display` variable.